### DISCUSSION TITLE
It is possible to add a "description" attribute to the VideoSearchResult class

### DISCUSSION THREAD

**Discussion by Ts-Pytham** [Tyrrrz/YoutubeExplode#544] [General]
Hi there, today I updated the library to the latest version and I found big changes and I found a new class (VideoSearchResult) but the new class doesn't have some attributes, my question is, is it possible to get these attributes without using another call?

**Comment by Tyrrrz** [comment]
The response returned by search does include description, but it's only a short snippet without any formatting (much like what you see when you search for videos vs. what you see on the video watch itself), so I decided not to include it because you'll get a completely different description when you request video metadata individually.

**Reply by Ts-Pytham** [reply]
I understand, in the beta version you used Video to get a set of videos, I am using that class, but if I want to get the description, I think I would have to use youtubeClient.Videos.GetAsync() and pass the id of the video and get the description, but wouldn't it be a bit slower? What do you recommend?

**Reply by Tyrrrz** [reply]
Yes, it will be slower, but unfortunately that's the only way

**Reply by Ts-Pytham** [reply]
Oh, right. Thanks
